### ISSUE TITLE
Any recommendation about a `i18n` library that work throughout the `apps` and `packages`?

### ISSUE THREAD

**Issue by 413n** [t3-oss/create-t3-turbo#116]
### Describe the feature you'd like to request

In my use case, I need a `i18n` lib with support of `json` files that contains the translations for each language.
Currently I'm using `lingui` only in the `nextjs` app, but I will need the translations also in the `packages/api` code.

### Describe the solution you'd like to see

Any recommendations?

**Comment by juliusmarminge** [comment]
Not familiar with lingui myself but seems to support all platforms according to their readme

**Comment by PiotrekPKP** [comment]
> 

Actually I was gonna ask about i18n... I see a lot of issues / comments / questions about it, there seems to be a lot of boilerplate going on with setting that up (typesafe `t` methods, file reading, language managing and stuff like that). Don't you think it's a great idea to introduce something to t3-app or t3-turbo? Of course not as a default addition, I would suggest adding a checkbox for it 🤔

**Comment by Bekacru** [comment]
Why not create a package and export the json files? Make the key the English phrase (or the language you'll use inside your code) and the value the translation for each lang in the json file. Next, create a component for both next and expo that will read the user-selected language from wherever it is placed before displaying the mapped value.  You can use <P> </P> for next and <AppText> </AppText> for expo. 
Here is an example component I use for Expo.
![code](https://user-images.githubusercontent.com/86073083/212864523-aa6fab62-cafd-4d14-9706-47a56e8e3d8f.png)

**Comment by 413n** [comment]
FYI, I solved my problem creating a new package that exports i18n and extracted all the locales in that package. Then I reuse that package where I need the translations. The only additional thing that I needed to do for tRPC was to catch the user language in the `context` and consequently `activate` that language. It all works smoothly!
(I can provide a snippet if anybody needs it to have more details)

I will close this since was all about a recommendation.
I suppose the feature request of the i18n's integration inside `create-t3-turbo` will need another issue.
